dev/gpctl/cmd/public-api.go

```diff
@@ -72,36 +69,3 @@ func newPublicAPIConn() (*grpc.ClientConn, error) {
 
 	return conn, nil
 }
-
-type connectHttpTransport struct {
```

Comment:
I had to introduce this last Friday to make gpctl+connect work, happy to see it go out so soon again :smile:

Reply:
@geropl Could you change owner files that we don't bother engine with change to files started with public-api*?

Reply:
@iQQBot if you can help again 🙏